include/SDL3_image/SDL_image.h

```diff
@@ -2403,6 +2403,14 @@ extern SDL_DECLSPEC IMG_Animation * SDLCALL IMG_LoadGIFAnimation_IO(SDL_IOStream
  */
 extern SDL_DECLSPEC IMG_Animation * SDLCALL IMG_LoadWEBPAnimation_IO(SDL_IOStream *src);
 
+#define IMG_PROP_ANIMATION_IGNORE_PROPS_BOOLEAN                 "SDL_image.animation.ignore_props"
+#define IMG_PROP_ANIMATION_DESCRIPTION_STRING                   "SDL_image.animation.description"
```

Comment:
```suggestion
#define IMG_PROP_METADATA_DESCRIPTION_STRING                   "SDL_image.metadata.description"
```
This will eventually be supported on single-image load/save, so the naming should reflect that.